tortoisebeacon/weakcoin/weak_coin.go

```diff
@@ -106,20 +120,27 @@ func (wc *weakCoin) Get(epoch types.EpochID, round types.RoundID) bool {
 }
 
 func (wc *weakCoin) PublishProposal(ctx context.Context, epoch types.EpochID, round types.RoundID) error {
-	p, err := wc.generateProposal(epoch, round)
+	wc.Log.With().Info("Publishing proposal",
+		log.Uint64("epoch_id", uint64(epoch)),
```

Comment:
nit: epoch was changed to uin32

Reply:
I missed that change. Then this needs to be fixed